src/screens/FeedScreen/index.js

```diff
@@ -356,6 +358,7 @@ const FeedScreen = (props) => {
           });
           fetchNextFeeds(momentumEvent);
         }}
+        snapToOffsets={[...Array(feeds.length).keys()].map((i) => i * contentHeight)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This line calculates the `snapToOffsets` property dynamically based on the number of feeds. While this approach is more flexible, it could potentially lead to performance issues if the number of feeds is very large. Consider implementing a limit or pagination to prevent potential performance degradation.
```diff
- snapToOffsets={[...Array(feeds.length).keys()].map((i) => i * contentHeight)}
+ snapToOffsets={[...Array(Math.min(feeds.length, MAX_FEEDS)).keys()].map((i) => i * contentHeight)}
```
In the above code, `MAX_FEEDS` should be a predefined constant that represents the maximum number of feeds you want to process at once.


<!-- This is an auto-generated comment by OSS CodeRabbit -->